packages/patternlab/styleguide/source/assets/js/modules/mapLeaflet.js

```diff
@@ -69,7 +69,7 @@ export default (function (window,document) {
 
       // custom marker icon 
       var markerIcon = L.icon({
-        iconUrl: `${ma.iconPath}marker-blue.svg`,
+        iconUrl: `${ma.iconPath}/marker-blue.svg`,

```

Comment:
```suggestion
        iconUrl: `${ma.iconPath}marker-blue.svg`,

```
fix marker in patternlab